server/build.gradle

```diff
@@ -81,6 +81,12 @@ dependencies {
 
 }
 
+spotless {
+  java {
+    targetExclude "src/main/java/org/elasticsearch/index/IndexVersion.java"
```

Comment:
Why can we not use spotless on this file?

Reply:
The constant lines are past the 140 char limit, so any auto-generated lines will immediately fail spotless (we had this problem in Version too, turned it off there). Whilst we work out what we want to do in this file, best if its off for now.